lib/active_model_serializers/callbacks.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from
+# https://github.com/rails/rails/blob/7f18ea14c8/activejob/lib/active_job/callbacks.rb
+require 'active_support/callbacks'
+
+module ActiveModelSerializers
+  # = ActiveModelSerializers Callbacks
+  #
+  # ActiveModelSerializers provides hooks during the life cycle of serialization and
+  # allow you to trigger logic. Available callbacks are:
+  #
+  # * <tt>around_render</tt>
+  #
+  module Callbacks
```

Comment:
I thought it made the loggng file easier to use and understand . File and organization based on active job b so seems fine to me

B mobile phone

> On Nov 10, 2015, at 8:04 AM, Mauro George <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/callbacks.rb:
> 
> > @@ -0,0 +1,55 @@
> > +# Adapted from
> > +# https://github.com/rails/rails/blob/7f18ea14c8/activejob/lib/active_job/callbacks.rb
> > +require 'active_support/callbacks'
> > +
> > +module ActiveModelSerializers
> > +  # = ActiveModelSerializers Callbacks
> > +  #
> > +  # ActiveModelSerializers provides hooks during the life cycle of serialization and
> > +  # allow you to trigger logic. Available callbacks are:
> > +  #
> > +  # * <tt>around_render</tt>
> > +  #
> > +  module Callbacks
> Since it just not much functionality. I vote to keep the things simple and remove the callback. But apparently all maintainers are ok with that, I can live with that.
> 
> —
> Reply to this email directly or view it on GitHub.
>